dapr/proto/runtime/v1/dapr.proto

```diff
@@ -217,6 +217,23 @@ message PublishEventRequest {
   // metadata property:
   // - key : the key of the message.
   map<string,string> metadata = 5;
+
+
+  // The retry strategy (optional).
+  //enum RetryStrategy {
```

Comment:
Keeping this enum should be OK but probably in the common proto.

@pkedy This PR would introduce a way to override the retry settings per request. Can you, please, review this too?